File: 백준/Bronze/28455. Union Maplestory/Union Maplestory.py
import sys
input = lambda: sys.stdin.readline().rstrip()


def main():
    N = int(input())
    L = [int(input()) for _ in range(N)]
    L.sort(reverse=True)
    lvs = [60,100,140,200,250]
    hap, up = 0, 0
    for l in L[:42]:
        for lv in lvs:
            if l>=lv:
                up += 1
        hap += l
    print(hap, up)


if __name__ == '__main__':
    main()

File: 백준/Bronze/28455. Union Maplestory/Union Maplestory.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int[] L = new int[N];
        for (int i = 0; i < N; i++) {
            L[i] = sc.nextInt();
        }
        Arrays.sort(L);
        for (int i = 0; i < N / 2; i++) {
            int temp = L[i];
            L[i] = L[N - i - 1];
            L[N - i - 1] = temp;
        }
        int[] lvs = {60, 100, 140, 200, 250};
        int hap = 0, up = 0;
        for (int i = 0; i < 42; i++) {
            if (i >= N) break;
            for (int lv : lvs) {
                if (L[i] >= lv) {
                    up++;
                }
            }
            hap += L[i];
        }
        System.out.println(hap + " " + up);
    }
}